Formula/pocl.rb

```diff
@@ -0,0 +1,27 @@
+class Pocl < Formula
+  desc "MIT-licensed open source implementation of the OpenCL standard"
+  homepage "http://portablecl.org/"
+  url "http://portablecl.org/downloads/pocl-1.0.tar.gz"
+  sha256 "94bd86a2f9847c03e6c3bf8dca12af3734f8b272ffeacbc3fa8fcca58844b1d4"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  # POCL 1.0 does not support LLVM 6
+  depends_on "llvm@5"
```

Comment:
We are typically reluctant to accept new formulas that depend on older software versions :(

Reply:
@fxcoudert How is LLVM 5 *older*?  It is the latest LLVM version supported by Homebrew today.  It seems that package is merely being defensive against a yet-to-be-supported future release of LLVM that is known to not work when built from source (because there are no publicly available LLVM 6 binaries for MacOS right now).

```
$ brew search llvm
==> Searching local taps...
llvm ✔                    llvm@3.7                  llvm@3.8                  llvm@3.9                  llvm@4
==> Searching taps on GitHub...
brew inf==> Searching blacklisted, migrated and deleted formulae...
$ brew info llvm
llvm: stable 5.0.1 (bottled), HEAD [keg-only]
Next-gen compiler infrastructure
https://llvm.org/
/usr/local/Cellar/llvm/5.0.1 (2,478 files, 1.2GB)
  Poured from bottle on 2018-01-03 at 11:19:26
```